### PR TITLE
Static deps & provides

### PR DESCRIPTION
**Pull request by jonathanwgoodwin**
@also @gchomatas @jhaber

### CODE REVIEW

**Comment by jhaber** [comment]
Do we need to include runtimeDeps or devDeps as well?

**Comment by jonathanwgoodwin** [comment]
Oh yeah. I'll add post lunch.

File: BlazarService/src/main/java/com/hubspot/blazar/discovery/hsstatic/HubSpotStaticModuleDiscovery.java [hunk 1]
```diff
@@ -60,21 +63,23 @@ public DiscoveryResult discover(GitInfo gitInfo) throws IOException {
     Set<DiscoveredModule> modules = new HashSet<>();
     Set<MalformedFile> malformedFiles = new HashSet<>();
     for (String config : staticConfigs) {
-      String moduleDirectory = String.valueOf(Paths.get(config).getParent());
-      String moduleName = moduleName(gitInfo, moduleDirectory);
+      String contents = gitHubHelper.contentsFor(config, repository, gitInfo);
+      StaticConfig configObject = objectMapper.readValue(contents, StaticConfig.class);
```
Comment (jhaber) [inline BlazarService/src/main/java/com/hubspot/blazar/discovery/hsstatic/HubSpotStaticModuleDiscovery.java]: We should add a malformed file if this parsing fails

**Comment by jonathanwgoodwin** [comment]
@jhaber We good here?

**Comment by jhaber** [comment]
cool lgtm

**Pull request merged**